pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -89,11 +89,9 @@ func Syncz(w http.ResponseWriter, _ *http.Request) {
 	for _, con := range adsClients {
 		con.mu.RLock()
 		if con.modelNode != nil {
-			proxyVersion, _ := con.modelNode.GetProxyVersion()
 			istioVersion, _ := con.modelNode.GetIstioVersion()
 			syncz = append(syncz, SyncStatus{
 				ProxyID:         con.modelNode.ID,
-				ProxyVersion:    proxyVersion,
```

Comment:
should maybe remove the field from the struct or comment on it that it's no longer used - probably want to keep it so istioctl is backwards compatible though